api/types/accesslist/accesslist.go

```diff
@@ -240,6 +240,10 @@ type Owner struct {
 	// Name is the username of the owner.
 	Name string `json:"name" yaml:"name"`
 
+	// Title is the title of an owner if it is of type MEMBERSHIP_KIND_LIST.
+	// This is only populated by the proxy when fetching an access list and its members for the web UI
+	Title string `json:"title" yaml:"title"`
```

Comment:
i need to add this value to the `Owner` similar to how i had to add it to `AccessListMember`. hijacking this PR to add this small change, but will make a separate commit if you want